LibNoDaveConnectionLibrary/DataTypes/Projectfolders/Step7V5/BlocksOfflineFolder.cs

```diff
@@ -323,8 +319,9 @@ private tmpBlock GetBlockBytes(ProjectBlockInfo blkInfo)
                         {
                             //DB Structure in Plain Text (Structure and StartValues!)
                             if (mc5code != null)
-                                myTmpBlk.blkinterface =
-                                    Project.ProjectEncoding.GetString(mc5code);
+                                if (!myTmpBlk.IsInstanceDB ) //only take interface if the block is not an instance DB. If it is, then reather take the interface from the FB declaration

```

Comment:
Do not overwrite the DB declartion if it was already obtained from the Function Block